public/index.html

```diff
@@ -18,6 +18,8 @@
       href="%PUBLIC_URL%/images/apple-touch-icon.png"
     />
     <meta name="msapplication-TileColor" content="#da532c" />
+    <!-- This polyfill is to fix https://helsinkisolutionoffice.atlassian.net/browse/TH-879 -->
+    <script src="https://cdn.jsdelivr.net/npm/text-encoding@0.7.0/lib/encoding.min.js"></script>
```

Comment:
what was the error in Edge?

Reply:
Console shows `TextEncoder is not defined` error without this polyfill. 

<img width="703" alt="Screenshot 2020-11-18 at 14 49 15" src="https://user-images.githubusercontent.com/24706814/99532626-46de8000-29ad-11eb-9b10-5d1b453fbe39.png">